src/Microsoft.VisualStudio.ProjectSystem.Managed/IO/SimpleFileWatcher.cs

```diff
@@ -52,23 +51,23 @@ public SimpleFileWatcher()
         /// </summary>
         public void Dispose()
         {
-            if (FileWatcher != null)
+            if (_fileWatcher != null)
             {
-                FileWatcher.EnableRaisingEvents = false;
+                _fileWatcher.EnableRaisingEvents = false;
                 if (_handler != null)
                 {
-                    FileWatcher.Created += _handler;
-                    FileWatcher.Deleted += _handler;
-                    FileWatcher.Changed += _handler;
+                    _fileWatcher.Created += _handler;
```

Comment:
I know this isn't your code, but while you're here fixing the property these seem like they should probably be `-=`